L/libsigcpp/libsigcpp@2.12.0/build_tarballs.jl

```diff
@@ -23,7 +23,7 @@ ninja install
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = expand_cxxstring_abis(supported_platforms(; experimental=true))
+platforms = expand_cxxstring_abis(supported_platforms(; experimental=true); skip=Returns(false))
```

Comment:
Min nit-pick: `experimental` does nothing nowadays, it's kept only for compatibility
```suggestion
platforms = expand_cxxstring_abis(supported_platforms(); skip=Returns(false))
```

Reply:
Good to know, thanks :+1: